tests/wpt/metadata/websockets/cookies/third-party-cookie-accepted.https.html.ini

```diff
@@ -0,0 +1,3 @@
+[third-party-cookie-accepted.https.html]
```

Comment:
This new test failure shows up because tungstenite correctly passes through multiple Set-Cookie headers in the WS response, whereas ws-rs appears to silently drop all but the first.